analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyCollectionsShouldNotBeEnumerated.cs

```diff
@@ -0,0 +1,457 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+
+namespace Tests.Diagnostics
+{
```

Comment:
Should go away (in this PR - easy to review). It's just a useless scaffolding noise
```suggestion
```